src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -61,8 +61,8 @@ public void invalidSearchPhraseDoesntTriggerSuggestionAndSearchCanBeCancelled()
     navigation.cancelSearch();
   }
 
-  @Test(groups = "navigation-navigateToPageUsingLocalNavigation")
-  public void navigateToPageUsingLocalNavigation() {
+  @Test(groups = "mercury_navigation_navigateToPageUsingLocalNavigation")
+  public void mercury_navigation_navigateToPageUsingLocalNavigation() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)